manifests/m/Microsoft/xstudio/1.7.1/Microsoft.xstudio.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate
+
+PackageIdentifier: Microsoft.xstudio
+PackageVersion: 1.7.1
+InstallerType: nullsoft
+FileExtensions:
+- svip
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/oxygen-dioxide/Zhihu-UWP/releases/download/v1/XStudioSinger_installer_1.7.1.exe

```

```suggestion
  InstallerUrl: https://singingtool.blob.core.chinacloudapi.cn/singingtool-container/XStudioSinger_2021-06-11_1.7.1.zip
```
This is the official download link from https://singer.xiaoice.com. You'll need to wait for `.zip` support to be added to the Windows Package Manager.